### PR TITLE
Enable hmr

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
Investigating solving https://github.com/blackbaud/skyux2/issues/406.  To test, install this branch and run `skyux serve --hmr` . We could also evaluate this being the default behavior.  Whichever is decided, a docs change would be necessary.

### CODE REVIEW

File: test/cli-serve.spec.js [hunk 1]
```diff
@@ -61,6 +61,40 @@ describe('cli serve', () => {
     }, webpackDevServer);
   });
 
+  it('should prepent hrm url to entries if devServer.hot is set', (done) => {
```
Comment (Blackbaud-SteveBrush) [inline test/cli-serve.spec.js]: "prepent"

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-BobbyEarl I tested this with SKY UX and it works great! My vote would be to make it the default behavior. If not, it would be nice to have a log or something to indicate that HMR is active.

**Comment by Blackbaud-BobbyEarl** [comment]
@Blackbaud-SteveBrush I added a log when hmr is enabled.  There's also already a log in the browser when HMR is enabled.

@Blackbaud-PaulCrowder any opinion on enabling HMR by default?

**Comment by Blackbaud-BobbyEarl** [comment]
I vote we leave this opt-in.  I'd like to get feedback on usage.  If it works well and developer's like it, I think we can introduce it as the default behavior in the next major release.

**Pull request merged**